packages/css/src/heading/README.md

```diff
@@ -0,0 +1,20 @@
+# Heading
+
+Een gebruiker zou middels headings een goede indruk moeten krijgen van de content van een webpagina. Een heading zegt iets over de onderstaande content.
+
+## Toegankelijkheid
+
+Headings en de heading structuur zijn vooral belangrijk voor screen readers en zoekmachines.
+
+## Richtlijnen
+
+- Één `h1` tag per pagina die het hoofdonderwerp van de pagina beschrijft.
+- Een heading omschrijft de content eronder.
+- Een heading niet gebruiken voor opmaak en styling.
+- Gebruik heading hierarchisch en sla niet een heading niveau over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
+
```

Comment:
Do we want to add the relevant WCAG rules here as well?

Reply:
Good one